### PR TITLE
Allow manual mode only.

### PR DESCRIPTION
**Pull request by m42e**
This will always tell the thermostat to keep manual mode. As
HomeBridge/Siri shall controll all changes to the Temperature.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "homebridge-eq3ble",
-  "version": "1.3.2",
+  "version": "1.3.3",
```
Comment (maxnowack) [inline package.json:3]: Please don't increment the version by yourself

File: src/index.js [hunk 2]
```diff
@@ -9,13 +9,15 @@ function thermostatFactory({ Service, Characteristic }) {
...
       this.address = config.address.toLowerCase()
+      this.manualOnly = config.manual || false
       this.discoverTimeout = config.discoverTimeout || (60 * 1000) // 1 minute
       this.connectionTimeout = config.connectionTimeout || (10 * 1000) // 10 seconds
       this.disableBoostSwitch = false
       this.temperatureDisplayUnits = Characteristic.TemperatureDisplayUnits.CELSIUS
 
       this.thermostat = new Thermostat({
         address: this.address,
+        manual: this.manualOnly,
```
Comment (maxnowack) [inline src/index.js:20]: Sometimes you're using `manual` and sometimes `manualOnly`. Please unify the property names. I think `manualOnly`is more clear.

**Review by maxnowack — CHANGES_REQUESTED** [review]

**Comment by joshhornby** [comment]
@maxnowack If I make the requested changes in a new PR would you accept?

**Comment by maxnowack** [comment]
@joshhornby sure!

**Comment by joshhornby** [comment]
@maxnowack Added a new PR https://github.com/maxnowack/homebridge-eq3ble/pull/20

**Pull request closed without merging**